### PR TITLE
added changes to documentation of print function in sorting.cpp

### PR DESCRIPTION
**Pull request by mnswdhw**
Manas Wadhwa
11940670

### CODE REVIEW

File: Sorting/sorting.cpp [hunk 1]
```diff
@@ -20,7 +20,12 @@ void Sort::GetInput(){
 }
 
 /**
- * Print.
+ * Depending upon the boolean input, prints the corresponding array separated by comma.When the input is true
```
Comment (shashwatj07) [inline Sorting/sorting.cpp]: ```commas. When```

File: Sorting/sorting.cpp [hunk 2]
```diff
@@ -20,7 +20,12 @@ void Sort::GetInput(){
...
+ * it prints the sorted array and when it is false it prints the unsorted array.
+ * 
+ * Inputs: boolean value True or False
```
Comment (shashwatj07) [inline Sorting/sorting.cpp]: ``` boolean, true if the sorted array is to be printed, false for unsorted ```

**Review by shashwatj07 — CHANGES_REQUESTED** [review]
Some items.

**Comment by yatendra1999** [comment]
Contains the same information in description and the inputs.

File: Sorting/sorting.cpp [hunk 3]
```diff
@@ -20,7 +20,12 @@ void Sort::GetInput(){
 }
 
 /**
- * Print.
+ * Depending upon the boolean input, prints the corresponding array separated by commas. When the input is true
+ * it prints the sorted array and when it is false it prints the unsorted array.
```
Comment (shashwatj07) [inline Sorting/sorting.cpp:24]: ```Array Printing Utility```

**Review by shashwatj07 — CHANGES_REQUESTED** [review]
Squash your commits into one.

**Comment by shashwatj07** [comment]
@mnswdhw ping

**Comment by shashwatj07** [comment]
@mnswdhw ping

**Pull request closed without merging**